src/components/item/publish/PublicationAttributeContainer.tsx

```diff
@@ -36,12 +36,12 @@ type Props = {
   dataTestId: string;
   title: string;
   titleIcon?: JSX.Element;
-  titleActionBtn: JSX.Element;
+  titleActionBtn?: JSX.Element;
   content?: JSX.Element | JSX.Element[];
   isLoading?: boolean;
-  emptyDataMessage: string;
-  attributeDescription: string;
-  onEmptyClick: () => void;
+  emptyDataMessage?: string;
+  attributeDescription?: string;
+  onEmptyClick?: () => void;
```

Comment:
Regarding openness of the props, I would propose to refactor in another PR where users of this component also define empty and filled component eg:
```ts
// LicenseContainer.tsx

content={hasNoData ? buildEmptyPlaceHolder() : content}
tooltip={hasNoData && <Warning ... />}
...
```

Reply:
Yes looks good, can you open an issue for that ?